### PR TITLE
Hannah tuttle

### PR DESCRIPTION
**Pull request by hannahtuttle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # dvs-website
 
 Pitch: Financial insecurity is the No. 1 reason survivors of domestic violence remain in or return to abuse. In order to break the cycle of violence, we must remove obstacles to financial security for survivors. To that end, FreeFrom (https://www.freefrom.org/) is developing an interactive calculator for their website. It will serve survivors to enable them to better understand the costs of leaving and staying safe. The calculator will include prompts to help users understand and identify the relevant costs. This project will not only help survivors and raise awareness of the costs of domestic violence, but will also capture valuable data that will help FreeForm drive positive policy change!
+
```
Comment (workingjubilee) [inline README.md]: Ah, nice catch on the spacing issues!

File: less/about.less [hunk 2]
```diff
@@ -0,0 +1,23 @@
+body{
+    margin: 0 auto;
+    max-width: 1000px;
+    .UI-Devs {
+        display: flex;
+        justify-content: space-between;
+
+    .dev {
+        border: 1px solid red;
```
Comment (workingjubilee) [inline less/about.less]: Ah, it's good to see you again, my old friend, `border: 1px solid red;`

File: less/about.less [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        flex-basis: column;
+        width:30%;
+        justify-content: center;
+        height: 300px;
```
Comment (workingjubilee) [inline less/about.less]: You have a good eye for where it is best to use relative-to-screen measurements vs. where more fixed ones might be better.

File: About/about.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+                </div>
+                <div class="dev">
+                    <h3>Neha Patel</h3>
+                    <p></p>
+                    <img class= "individual-photos">
+                </div>
+                <div class="dev">
+                    <h3>Hannah Tuttle</h3>
+                    <p>Hannah is a wife and mom of twin 2 year old girls. She is currently studying Full stack Web Development ar Lambda School.</p>
+                    <img class= "individual-photos" src="../img/IMG_0048 2.jpg">
```
Comment (workingjubilee) [inline About/about.html]: You might want to standardize on an image naming or folder convention. It's fine if images have names that would be nonsensical outside of the project, mind, it just helps if the convention is internally consistent, whatever helps you find the next image or guess the contents of it from the filename or folder location.

File: less/about.less (hunk 2 above)
Comment (HawkeDracorr) [reply less/about.less]: May or may not be helpful in the future, but I use `outline: red solid 1px;`, which does the same thing, but doesn't take space on the page. It kinda exists on a separate layer. Useful if you have issues with the box model.

File: About/about.html [hunk 5]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+	<head>
+            <link rel="stylesheet" href="../css/index.css">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	</head>
+	<body>
+		<nav>
+                <img src="../img/logo.png" alt="Freefrom Logo">
+                <div class="nav-items">
```
Comment (HawkeDracorr) [inline About/about.html]: I've updated the headings and nav items to be all caps to closer match the design of the Freefrom site. It should be pulled in with your next pull request.

File: About/about.html [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+            <h2>UI Developers</h2>
+        
+            <div class="UI-Devs">  
+                <div class="dev dev1">
+                    <div class ="name-description">
+                    <h3>Hawke Draccor</h3>
+                    <p>Lorem ipsum dolor sit amet, ei error iudico maluisset vim, mel odio elit abhorreant ad. At probo hendrerit nam, eirmod vidisse sensibus has in, ex his iisque ullamcorper. Ea vel delicata tincidunt, in nihil iuvaret pri. Dolore doming mediocrem nec eu.</p>
+                    </div>
+                    <div class= "photo">
+                    <img class= "photo1" src="../img/Mindy.jpg">
```
Comment (HawkeDracorr) [inline About/about.html]: I actually have no headshot, but will provide you with an image to use in it's place.

File: css/index.css [hunk 7]
```diff
@@ -152,6 +152,94 @@ nav .nav-items a {
 html {
   font-size: 62.5%;
 }
+body {
+  margin: 0 auto;
+  max-width: 1000px;
```
Comment (HawkeDracorr) [inline css/index.css]: I've transitioned away from absolute pixels on my end. For consistency, I used `rem` for non-text elements and `em` for text. 1rem/em = 10px, for reference (As it's set within the global less file, anyway.)

**Pull request merged**